packages/wdio-appium-service/src/launcher.ts

```diff
@@ -125,4 +146,18 @@ export default class AppiumLauncher {
         this.process.stdout.pipe(logStream)
         this.process.stderr.pipe(logStream)
     }
+
+    private getAppiumCommand (moduleName = 'appium') {
```

Comment:
moved from utils.js as a private member of AppiumLauncher class. It implies to be a part of this class and we can avoid additional import of logger.